schema.gql

```diff
@@ -0,0 +1,376 @@
+# ------------------------------------------------------
```

Comment:
Doesn't seem like this file belongs here? Isn't this already in `graphql-server`?